jekyll-export-cli.php

```diff
@@ -16,12 +16,54 @@
  *
  * Must be run in the wordpress-to-jekyll-exporter/ directory.
  */
-require '../../../wp-load.php';
+
+// Uncomment for extra replace options
+
+$wordpress_path = "../../../";
+//$wordpress_path = "/home/wordpress/";
+
+require $wordpress_path . 'wp-load.php';
 require_once 'jekyll-exporter.php'; // Ensure plugin is "activated".
 
+$config["site_url"] = "https://www.mywebsite.fr/";
```

Comment:
Is there a way to get the site URL without hard coding it into the plugin itself?

Reply:
I added a default value with get_site_url and the possibility to override it.